strawberry/types/types.py

```diff
@@ -119,10 +119,9 @@ def is_specialized_generic(self) -> bool:
         if not self.is_generic:
             return False
 
-        type_var_map = get_specialized_type_var_map(self.origin, include_type_vars=True)
-        return type_var_map is None or not any(
-            isinstance(arg, TypeVar) for arg in type_var_map.values()
-        )
+        type_var_map = get_specialized_type_var_map(self.origin)
+
+        return not type_var_map and not getattr(self.origin, "__parameters__", None)
```

Comment:
@bellini666 can you double check this? 😊

Reply:
Yeah, this seems correct.

Also, I tested this branch locally with this change:

```python
@property
def is_specialized_generic(self) -> bool:
    return self.is_generic and not getattr(self.origin, "__parameters__", None)
```

And all tests worked fine! So probably we don't actually need to check for `type_var_map` in here

Reply:
I'll update thanks!

do you also want to test a pre-release on your projects or shall we just ship the release? :D